node-tests/blueprints/dependencyScenarios.js

```diff
@@ -0,0 +1,586 @@
+'use strict';
+
+const bs3Regex = '\\^3\\.\\d+\\.\\d+';
+const bs4Regex = 'next';
+const scenarios = [
+  {
```

Comment:
Nice design! So this first one is equivalent to `installed: {}, options: {}`, correct? If you don't like the name idea below, we should consider being explicit on the empty definition sections.